### ISSUE TITLE
error with mobilenet in int8

### ISSUE THREAD

**Issue by qianqing13579** [ROCmSoftwarePlatform/AMDMIGraphX#685]
I want to use mobilenet(include depthwise convolution) in int8,but there is an error: MIGraphX/src/targets/gpu/quant_convolution.cpp:88: QUANT_CONVOLUTION: find convolution failed.

Here is the code in pytorch:
def conv_dw(inp, oup, stride, leaky=0.1):
    return nn.Sequential(
        nn.Conv2d(inp, inp, 3, stride, 1, groups=inp, bias=False),
        nn.BatchNorm2d(inp),
        nn.LeakyReLU(negative_slope= leaky,inplace=True),

        nn.Conv2d(inp, oup, 1, 1, 0, bias=False),
        nn.BatchNorm2d(oup),
        nn.LeakyReLU(negative_slope= leaky,inplace=True),
    )

class MobileNetV1(nn.Module):
    def __init__(self,num_classes=3):
        super(MobileNetV1, self).__init__()
        self.stage1 = nn.Sequential(
            conv_bn(3, 8, 2, leaky = 0.1),    # 3
            conv_dw(8, 16, 1),   # 7
            conv_dw(16, 32, 2),  # 11
            conv_dw(32, 32, 1),  # 19
            conv_dw(32, 64, 2),  # 27
            conv_dw(64, 64, 1),  # 43
        )
        self.stage2 = nn.Sequential(
            conv_dw(64, 128, 2),  # 43 + 16 = 59
            conv_dw(128, 128, 1), # 59 + 32 = 91
            conv_dw(128, 128, 1), # 91 + 32 = 123
            conv_dw(128, 128, 1), # 123 + 32 = 155
            conv_dw(128, 128, 1), # 155 + 32 = 187
            conv_dw(128, 128, 1), # 187 + 32 = 219
        )
        self.stage3 = nn.Sequential(
            conv_dw(128, 256, 2), # 219 +3 2 = 241
            conv_dw(256, 256, 1), # 241 + 64 = 301
        )
        
        self.avg = nn.AdaptiveAvgPool2d((1,1))
        self.fc = nn.Linear(256, num_classes)

    def forward(self, x):
        x = self.stage1(x)
        x = self.stage2(x)
        x = self.stage3(x)
        x = self.avg(x)
        # x = x.view(-1, 256)
        x = torch.flatten(x, 1)
        x = self.fc(x)
        return x

**Comment by pfultz2** [comment]
Do you have an onnx file for this? 

Initially, this looks like MIOpen doesnt support int8 for group convolutions. With an onnx file I can report this back to the MIOpen team.

**Comment by qianqing13579** [comment]
@pfultz2 ，here is the link: https://pan.baidu.com/s/1twW_tE0sJblvfG0sSTdiUQ , password: 8lrf

**Comment by causten** [comment]
Is this still an issue?
